cmd/scaffold.go

```diff
@@ -704,7 +704,9 @@ func (fnb *FlowNodeBuilder) initFvmOptions() {
 		fvm.WithChain(fnb.RootChainID.Chain()),
 		fvm.WithBlocks(blockFinder),
 		fvm.WithAccountStorageLimit(true),
-		fvm.WithTransactionFeesEnabled(true),
```

Comment:
I removed this line here, just to confirm this is correct @janezpodhostnik

Reply:
Yes this change is correct. it should be limited to `fnb.RootChainID == flow.Testnet || fnb.RootChainID == flow.Canary || fnb.RootChainID == flow.Mainnet` like it is below.